pkg/security/admission/admission.go

```diff
@@ -216,40 +240,51 @@ func (c *constraint) createProvidersFromConstraints(ns string, sccs []*kapi.Secu
 				continue
 			}
 
+			// Make a copy of the constraint so we don't mutate the store's cache
+			var constraintCopy kapi.SecurityContextConstraints = *constraint
+			constraint = &constraintCopy
```

Comment:
This is warping my mind a little.  Can we move it to the top of the loop, choose a non-conflicting name, and assign the final thing to `constraintCopy`?  Something like this:

```go
for _, scc := range sccs{
    var t kapi.SecurityContextConstraints = *scc
    sccCopy := &t
```

The name change would ensure you don't accidentally leave any stragglers.